[2024-11-06T19:48:06.390+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-06T19:48:06.419+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: SetupETLtables.createTable scheduled__2024-11-05T02:30:00+00:00 [queued]>
[2024-11-06T19:48:06.430+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: SetupETLtables.createTable scheduled__2024-11-05T02:30:00+00:00 [queued]>
[2024-11-06T19:48:06.432+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-06T19:48:06.451+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): createTable> on 2024-11-05 02:30:00+00:00
[2024-11-06T19:48:06.467+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'SetupETLtables', 'createTable', 'scheduled__2024-11-05T02:30:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/etl_table_snowflake.py', '--cfg-path', '/tmp/tmp_j480cpm']
[2024-11-06T19:48:06.472+0000] {standard_task_runner.py:91} INFO - Job 120: Subtask createTable
[2024-11-06T19:48:06.470+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=190) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-06T19:48:06.475+0000] {standard_task_runner.py:63} INFO - Started process 216 to run task
[2024-11-06T19:48:06.533+0000] {task_command.py:426} INFO - Running <TaskInstance: SetupETLtables.createTable scheduled__2024-11-05T02:30:00+00:00 [running]> on host 599a8e9540bd
[2024-11-06T19:48:06.623+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='SetupETLtables' AIRFLOW_CTX_TASK_ID='createTable' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T02:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-05T02:30:00+00:00'
[2024-11-06T19:48:06.625+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-06T19:48:06.634+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-11-06T19:48:06.635+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.4-linuxkit-aarch64-with-glibc2.36
[2024-11-06T19:48:06.638+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-06T19:48:07.738+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-06T19:48:07.918+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-06T19:48:07.918+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-06T19:48:07.921+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-06T19:48:07.946+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=SetupETLtables, task_id=createTable, run_id=scheduled__2024-11-05T02:30:00+00:00, execution_date=20241105T023000, start_date=20241106T194806, end_date=20241106T194807
[2024-11-06T19:48:08.019+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-06T19:48:08.042+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-06T19:48:08.044+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
